ocrdmonitor/server/workspaces/_listroutes.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from fastapi import APIRouter, Request, Response
+from fastapi.templating import Jinja2Templates
+
+from ocrdbrowser import workspace
+from ocrdmonitor.server.settings import OcrdBrowserSettings
+
+
+def register_listroutes(
+    router: APIRouter, templates: Jinja2Templates, browser_settings: OcrdBrowserSettings
+) -> None:
+    @router.get("/", name="workspaces.list")
+    def list_workspaces(request: Request) -> Response:
+        spaces = [
+            Path(space).relative_to(browser_settings.workspace_dir)
+            for space in workspace.list_all(browser_settings.workspace_dir)
```

Comment:
> It does work on other workspaces though.
> 
> Could it be that we should restrict the list of workspaces to those in the Manager's internal directory `./ocr-d`? The testdata-presentation above was only present on the Kitodo outside (and had no images present locally, only ALTO).

Indeed, that was the problem! 

But since we use `OCRD_BROWSER__WORKSPACE_DIR=/data` (i.e. `OcrdBrowserSettings.workspace_dir`) both as **starting point** for `list_all` here **and** as **reference point** for relative paths, we should be safe simply switching to `/data/ocr-d` (for both).